modules/core/src/main/java/org/apache/ignite/internal/processors/cluster/IGridClusterStateProcessor.java

```diff
@@ -108,30 +108,20 @@ boolean onStateChangeMessage(AffinityTopologyVersion topVer,
     /** */
     void cacheProcessorStarted();
 
-    /**
-     * @param activate New cluster state.
-     * @param baselineNodes New baseline nodes.
-     * @param forceChangeBaselineTopology Force change baseline topology.
-     * @return State change future.
-     * @deprecated Use {@link #changeGlobalState(ClusterState, Collection, boolean)} instead.
-     */
-    @Deprecated
-    IgniteInternalFuture<?> changeGlobalState(
-        boolean activate,
-        Collection<? extends BaselineNode> baselineNodes,
-        boolean forceChangeBaselineTopology
-    );
-
     /**
      * @param state New cluster state.
+     * @param force If {@code True} skips checking of the operation safety.
```

Comment:
We should rename this flag to `forceDeactivation` because it value applies to the deactivation command only.

Reply:
Fixed.